values.yaml

```diff
@@ -14,7 +14,7 @@ kobotoolbox:
   # - KPI Cache - 4
   # - Kobocat Cache - 5
   # - Service account - 6
-  redis: "redis://:pass@redis-master:6379"  # Do not add / nor /1 suffix
+  #redis: "redis://:pass@redis-master:6379"  # Do not add / nor /1 suffix
   #redisSession: "redis://:pass@redis-master:6379/1"
   #serviceAccountBackend: "redis://:pass@redis-master:6379/1"
   djangoLanguageCodes: "ar cs de-DE en es fr hi ku pl pt tr zh-hans"
```

Comment:
Once again, not from this PR, but languageCodes have changed.
FYI: https://github.com/kobotoolbox/kobo-install/pull/216/files

Reply:
Updated